ui/src/app/cohort-search/redux/actions/creators.ts

```diff
@@ -2,9 +2,16 @@
 import {
   BEGIN_CRITERIA_REQUEST,
   BEGIN_ALL_CRITERIA_REQUEST,
+  BEGIN_DRUG_CRITERIA_REQUEST,
   LOAD_CRITERIA_RESULTS,
   CANCEL_CRITERIA_REQUEST,
   SET_CRITERIA_SEARCH,
+  BEGIN_AUTOCOMPLETE_REQUEST,
```

Comment:
Is this reusable for all autocomplete request? or is it specific to meds? If specific to meds should rename.